src/docs/src/ddocs/mango.rst

```diff
@@ -0,0 +1,922 @@
+            "$lte": 1903
+        },
+        "$not": {
+            "year": 1901
+        }
+    }
+
+.. _find/nor:
+
+The ``$nor`` operator
+~~~~~~~~~~~~~~~~~~~~~
+
+The ``$nor`` operator matches if the given selector does not match. Below is an
+example used with an index on the field ``"year"``:
+
+.. code-block:: javascript
+
+    {
+        "year": {
+            "$gte": 1900.
+            "$lte": 1910
+        },
+        "$nor": [
+            { "year": 1901 },
+            { "year": 1905 },
+            { "year": 1907 }
+        ]
+    }
+
+.. _find/all:
+
+The ``$all`` operator
+~~~~~~~~~~~~~~~~~~~~~
+
+The ``$all`` operator matches an array value if it contains all the elements of
+the argument array. Below is an example used with the primary index
+(``_all_docs``):
+
+.. code-block:: javascript
+
+    {
+        "_id": {
+            "$gt": null
+        },
+        "genre": {
+            "$all": ["Comedy","Short"]
+        }
+    }
+
+.. _find/elemmatch:
+
+The ``$elemMatch`` operator
+~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+The ``$elemMatch`` operator matches and returns all documents that contain an
+array field with at least one element matching the supplied query criteria.
+Below is an example used with the primary index (``_all_docs``):
+
+.. code-block:: javascript
+
+    {
+        "_id": { "$gt": null },
+        "genre": {
+            "$elemMatch": {
+                "$eq": "Horror"
+            }
+        }
+    }
+
+.. _find/allmatch:
+
+The ``$allMatch`` operator
+~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+The ``$allMatch`` operator matches and returns all documents that contain an
+array field with all its elements matching the supplied query criteria. Below
+is an example used with the primary index (``_all_docs``):
+
+.. code-block:: javascript
+
+    {
+        "_id": { "$gt": null },
+        "genre": {
+            "$allMatch": {
+                "$eq": "Horror"
+            }
+        }
+    }
+
+.. _find/keymapmatch:
+
+The ``$keyMapMatch`` operator
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+The ``$keyMapMatch`` operator matches and returns all documents that contain a
+map that contains at least one key that matches all the specified query criteria.
+Below is an example used with the primary index (``_all_docs``):
+
+.. code-block:: javascript
+
+    {
+        "_id": { "$gt": null },
+        "cameras": {
+            "$keyMapMatch": {
+                "$eq": "secondary"
+            }
+        }
+    }
+
+.. _find/text:
+
+The ``$text`` operator
+~~~~~~~~~~~~~~~~~~~~~~
+
+The ``$text`` operator performs a text search using either a search or nouveau
+index. The specifics of the query follow either
+:ref:`search syntax <ddoc/search/syntax>` or
+:ref:`nouveau syntax <ddoc/nouveau/syntax>` (which both use Lucene and implement
+the same syntax).
+
+.. code-block:: javascript
+
+    {
+        "_id": { "$gt": null },
+        "cameras": {
+            "$keyMapMatch": {
+                "$eq": "secondary"
+            }
```

Comment:
this code example looks copy/pasted from the previous one and should show a `$text` operator instead.

Reply:
Oh, yeah, I did copy/paste this and forgot to update the example.